docs/api/openapi_detector_api.yaml

```diff
@@ -229,23 +226,16 @@ components:
   schemas:
     ChatAnalysisHttpRequest:
       properties:
-        chat_history:
-          items:
-            $ref: '#/components/schemas/Message'
+        messages:
+          title: Chat Messages
           type: array
-          title: Chat History
-          example:
-            - content: You are a helpful assistant.
-              author: system
-            - content: Hi, is this powered by siri or alexa?
-              author: user
-            - content: Better, it's watsonx
-              author: assistant
-            - content: This is awesome!
-              author: user
+          items:
+            allOf:
+              - $ref: https://raw.githubusercontent.com/openai/openai-openapi/master/openapi.yaml#/components/schemas/ChatCompletionRequestMessage
```

Comment:
hmm. this would include `name` etc as well. Just want to make a note of that. I don't think there is any harm in that.